pennylane/ops/op_math/composite.py

```diff
@@ -207,7 +207,6 @@ def overlapping_ops(self) -> list[list[Operator]]:
             while i < len(groups):
                 if first_group_idx is None and any(wire in op.wires for wire in groups[i][1]):
```

Comment:
Not a blocking comment: why do we do:
```python
groups[i][1]
```

instead of 

```python
groups[I][0] 
```
?

Reply:
Ah I see, each of them is a tuple `(op, wires)` right?

Reply:
what does `groups[I][0]` mean?

Reply:
For each group `group[0]` are the ops, and `group[1]` are the wires.